moveit_kinematics/ikfast_kinematics_plugin/README.md

```diff
@@ -8,7 +8,3 @@ Generates a IKFast kinematics plugin for MoveIt using [OpenRave](http://openrave
 Tested on ROS Hydro and greater with Catkin using OpenRave 0.8 with a 6dof and 7dof robot arm manipulator. Does not work with greater than 7dof.
 
 [Documentation on docs.ros.org](https://ros-planning.github.io/moveit_tutorials/doc/ikfast/ikfast_tutorial.html#creating-the-ikfast-moveit-plugin)
-
-## Build Status
-
-indigo-devel branch: [![Build Status](https://travis-ci.org/ros-planning/moveit_ikfast.png?branch=indigo-devel)](https://travis-ci.org/ros-planning/moveit_ikfast)
```

Comment:
Is this necessary? Looks like build status is captured in the main MoveIt2 readme.
If it is necessary, this needs an update, as the TravisCI link is dead.